tests/jlm/mlir/TestJlmToMlirToJlm.cpp

```diff
@@ -640,6 +640,69 @@ TestFpBinary()
 }
 JLM_UNIT_TEST_REGISTER("jlm/mlir/TestMlirFpBinaryGen", TestFpBinary)
 
+static void
+TestFMulAddOp()
+{
+  using namespace jlm::llvm;
+  using namespace mlir::rvsdg;
+
+  auto rvsdgModule = RvsdgModule::Create(jlm::util::FilePath(""), "", "");
+  auto graph = &rvsdgModule->Rvsdg();
+  {
```

Comment:
This compound statement is kind of useless, no?

Reply:
Yes it does not do anything, it was just copied from the other test I based mine on. I can remove it, or we can leave it for a later PR that cleans up all the tests if desired.

What do you think? @phate

Reply:
Your choice. I am fine either way.